lib/api/1.1/southbound/router.js

```diff
@@ -0,0 +1,37 @@
+// Copyright 2015, EMC, Inc.
```

Comment:
This `southbound/router.js` is very alike with the `nothbound/router.js` , the only difference should be router.use. If in future we have v2 api, we need create more this kind router. So whether we can create some common router factory and generate different routers by parameter?